lib/services/android-project-service.ts

```diff
@@ -181,35 +184,37 @@ class AndroidProjectService implements IPlatformProjectService {
 	public isPlatformPrepared(projectRoot: string): IFuture<boolean> {
 		return this.$fs.exists(path.join(projectRoot, "assets", constants.APP_FOLDER_NAME));
 	}
-
-	private parseProjectProperties(projDir: string, destDir: string): void {
-		let projProp = path.join(projDir, "project.properties");
-
-		if (!this.$fs.exists(projProp).wait()) {
-			this.$logger.warn("Warning: File %s does not exist", projProp);
-			return;
-		}
-
-		let lines = this.$fs.readText(projProp, "utf-8").wait().split(os.EOL);
-
-		let regEx = /android\.library\.reference\.(\d+)=(.*)/;
-		lines.forEach(elem => {
-			let match = elem.match(regEx);
-			if (match) {
-				let libRef: ILibRef = { idx: parseInt(match[1]), path: match[2].trim() };
-				libRef.adjustedPath = this.$fs.isRelativePath(libRef.path) ? path.join(projDir, libRef.path) : libRef.path;
-				this.parseProjectProperties(libRef.adjustedPath, destDir);
+	
+	private parseProjectProperties(projDir: string, destDir: string): IFuture<void> {
+		return (() => {
+			let projProp = path.join(projDir, "project.properties");
+	
+			if (!this.$fs.exists(projProp).wait()) {
+				this.$logger.warn("Warning: File %s does not exist", projProp);
+				return;
 			}
-		});
-
-		this.$logger.info("Copying %s", projDir);
-		shell.cp("-Rf", projDir, destDir);
-
-		let targetDir = path.join(destDir, path.basename(projDir));
-		// TODO: parametrize targetSdk
-		let targetSdk = "android-17";
-		this.$logger.info("Generate build.xml for %s", targetDir);
-		this.runAndroidUpdate(targetDir, targetSdk).wait();
+	
+			let lines = this.$fs.readText(projProp, "utf-8").wait().split(os.EOL);
```

Comment:
I think the correct spelling of encoding is utf8, see https://nodejs.org/api/fs.html#fs_fs_writefile_filename_data_options_callback